5_databases/week2/week2-hmw-part4.sql

```diff
@@ -0,0 +1,76 @@
+-- Active: 1706786442782@@127.0.0.1@3306@Investment_Bank
+CREATE DATABASE Investment_Bank
+    DEFAULT CHARACTER SET = 'utf8mb4';
+
+USE Investment_Bank;
+
+
+SELECT * FROM Client_Type;
+SELECT * FROM Registration_Number_Type;
+SELECT * FROM Instrument_Type;
+
+
+-- Create Client Type 
+CREATE TABLE Client_Type (
```

Comment:
Can a client have multiple client types?

Reply:
We decided on class that a client cannot have multiple client types. How can I change the table?

Reply:
you can just append the client type as a column to the client table and remove the client type table. that would be cleaner and easier to read. overall I like this use case. Nice job! thank you!